examples/bpk-component-price/examples.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import BpkPrice from '../../packages/bpk-component-price';
+
+const DefaultExample = () => <BpkPrice title="£1,830" />;
+
+const WithDescriptionExample = () => (
+  <BpkPrice title="£1,830" description="/ night" />
+);
+
+const WithSubtitleDescriptionExample = () => (
+  <BpkPrice subtitle="£209" title="£1,830" description="/ night" />
```

Comment:
For `description` i guess it would better only pass `night` and `/` in `BpkPrice` component.

Reply:
Thank you, Ezreal. I think one field is enough and more flexible.

Reply:
I agree with @Supremeyh if we always need to pass `/`, why not stick it in this component

Reply:
Thank you, Jason. Actually, in our usage. It won't be `/` now. It will be `a night` or `total` or `total stay` etc.
![image](https://user-images.githubusercontent.com/11376543/172085687-09ed9bd9-05d9-45c1-b734-b9d4c7643be5.png)
![image](https://user-images.githubusercontent.com/11376543/172086077-4543e63b-3bc4-4f65-a8b6-da13e2557f03.png)
![image](https://user-images.githubusercontent.com/11376543/172086138-44634dc8-b0c2-42ca-b699-748343df11bc.png)